tracktor/fulfillment/add-tags-of-tracktor-latest-delivery-status/mesa.json

```diff
@@ -0,0 +1,63 @@
+{
+  "key": "add_tags_of_tracktor_latest_delivery_status",
+  "name": "Tag Tracktor orders with current delivery status",
+  "version": "1.0.0",
+  "description": "Will not remove past Tracktor delivery status tags. ",
+  "video": "",
+  "readme": "",
+  "tags": [],
+  "source": "tracktor",
+  "destination": "shopify_api",
+  "enabled": false,
```

Comment:
Looks good!

I didn't see the need to configure any of the steps before enabling the workflow. Did you want to keep "enabled" as "false"?

Reply:
Good catch, thanks. I will make that change.